### PR TITLE
Fields for RSVP base fields

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#112960](http://central.tri.be/issues/112960)
---
This isn't done there are a few fields missing but should allow the creation of RSVP via the correct Rest EndPoint.

`POST` to `/wp-json/v2/wp/tribe_rsvp_attendees`, it's a base WordPress Post Endpoint so creation should follow the standards here: https://developer.wordpress.org/rest-api/reference/posts/#create-a-post

We are still missing going and not going, which I will add tomorrow.

### CODE REVIEW

File: plugins/tickets/src/Tribe/Meta.php [hunk 1]
```diff
@@ -0,0 +1,220 @@
+<?php
+/**
+ * Initialize Gutenberg Event Meta fields
+ *
+ * @since TBD
+ */
+class Tribe__Gutenberg__Tickets__Meta {
```
Comment (mitogh) [inline plugins/tickets/src/Tribe/Meta.php]: Wonder about this one is only for RSVP or `Tickets` as well because of the naming here mainly

File: plugins/tickets/src/Tribe/Meta.php [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+			$this->text()
+		);
+
+		register_meta(
+			'post',
+			Tribe__Tickets__Global_Stock::TICKET_STOCK_CAP,
+			$this->text()
+		);
+
+		// $woo = tribe( 'tickets-plus.commerce.woo' );
```
Comment (mitogh) [inline plugins/tickets/src/Tribe/Meta.php]: I guess this comment shouldn't be part of this PR?

File: plugins/tickets/src/Tribe/Meta.php (hunk 1 above)
Comment (bordoni) [reply plugins/tickets/src/Tribe/Meta.php]: `Tribe__Gutenberg__Tickets` is the naming for the Autoloader to find the class, but this class will most likely go away soon, in favor of figuring out a better way to handle the Rest API endpoint meta fields.

So for now I am giving a generic name just like in events

Comment (mitogh) [reply plugins/tickets/src/Tribe/Meta.php]: Makes sense, thank you!
